lib/reporters/mini.js

```diff
@@ -216,7 +216,16 @@ MiniReporter.prototype.finish = function (runStatus) {
 	}
 
 	if (runStatus.failFastEnabled === true) {
-		status += '\n\n  ' + colors.failFast('`--fail-fast` is on. Any number of tests may have been skipped');
+		status += '\n\n  ' + colors.information('`--fail-fast` is on. Any number of tests may have been skipped');
+	}
+
+	if (runStatus.hasExclusive === true) {
+		var remainingTests = runStatus.testCount - (runStatus.passCount + runStatus.failCount);
```

Comment:
This should be computed in `lib/run-status.js`, which @jarlehansen is working on in #1179. Note that as I commented there (https://github.com/avajs/ava/pull/1179#discussion_r95819806) you need to include other test states in this count, see the highlighted lines in https://github.com/avajs/ava/blob/033d4dcdcbdadbf665c740ff450c2a775a8373dc/lib/run-status.js#L41:L45.